applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -778,71 +780,74 @@ static int cmd_grammar_check(const uint8_t *cmd, uint16_t length)
 	return 0;
 }
 
-static void format_final_result(char *buf)
+static char *strrstr(const char *str1, const char *str2)
+{
+	size_t len1;
+	size_t len2;
+
+	if (str1 == NULL || str2 == NULL) {
+		return NULL;
+	}
+
+	len1 = strlen(str1);
+	len2 = strlen(str2);
+
+	if (len2 > len1 || len1 == 0 || len2 == 0) {
+		return NULL;
+	}
+
+	for (int i = len1 - len2; i >= 0; i--) {
+		if (strncmp(str1 + i, str2, len2) == 0) {
+			return (char *)str1 + i;
+		}
+	}
+
+	return NULL;
+}
```

Comment:
Why default library strstr -function does not work here?

Reply:
This is `str**r**str`, AKA find the last occurrence of a string in a string, which is not on the standard C library.